raft/src/test/java/org/apache/kafka/raft/KafkaRaftClientSnapshotTest.java

```diff
@@ -1977,12 +1978,20 @@ private static ReplicaKey replicaKey(int id, boolean withDirectoryId) {
         return ReplicaKey.of(id, directoryId);
     }
 
+    private static Integer randomReplicaId() {
+        int mockAddressPrefix = 9990;
+        // Number of nodes we can set up if we got a random number that is maximum in the range
+        int reservedNumberOfPorts = 50;
+        int maxPort = 65535 - mockAddressPrefix - reservedNumberOfPorts;
+        return ThreadLocalRandom.current().nextInt(maxPort);
+    }
```

Comment:
Let's return an `int` instead of the object `Integer`: `private static int randomReplicaId()`.

The variable names `mockAddressPrefix` and `reserverdNumberOfPorts` are misleading this id has nothing to do with those concepts. If you want to limit the range of the integer, I am okay with generating a random id from 0 to 1024.

Reply:
Thanks for the feedback. Updated to generate from 0 - 1024